src/components/external-source/ExternalSourcesPanel.svelte

```diff
@@ -0,0 +1,185 @@
+  $: filteredDerivationGroups.forEach(group => {
+    // ...and repopulate it every time the links change. this handles deletion correctly
+    if (group.source_type_name) {
+      if (
+        mappedDerivationGroups[group.source_type_name] &&
+        !mappedDerivationGroups[group.source_type_name]
+          .map(mappedDerivationGroup => mappedDerivationGroup.name)
+          .includes(group.name)
+      ) {
+        // use string later for source type
+        mappedDerivationGroups[group.source_type_name]?.push(group);
+      } else {
+        mappedDerivationGroups[group.source_type_name] = [group];
+      }
+    }
+  });
+
+  function onManageDerivationGroups() {
+    effects.managePlanDerivationGroups(user);
+  }
+</script>
+
+<Panel>
+  <svelte:fragment slot="header">
+    <GridMenu {gridSection} title="External Sources" />
+  </svelte:fragment>
+
+  <svelte:fragment slot="body">
+    <CollapsibleListControls
+      placeholder="Filter External Sources"
+      on:input={event => (filterText = event.detail.value)}
+    >
+      <svelte:fragment slot="right">
+        <button
+          name="manage-derivation-groups"
+          class="st-button active"
+          on:click|stopPropagation={onManageDerivationGroups}
+          use:tooltip={{ content: 'Select derivation groups to associate with this plan', placement: 'top' }}
+        >
+          Manage Derivation Groups
+        </button>
+      </svelte:fragment>
+    </CollapsibleListControls>
+
+    <AlertError class="m-2" error={$derivationGroupPlanLinkError} />
+
+    {#if unseenSources.length || unseenDeletedSources.length}
+      <div style="padding-top: 10px">
+        <CardList>
+          {#if unseenSources.length}
+            <ExternalSourceUpdateCard
+              deleted={false}
+              sources={unseenSources}
+              on:dismiss={() => {
+                effects.createExternalSourceSeenEntry(unseenSources, user);
+              }}
+            />
+          {/if}
+          {#if unseenDeletedSources.length}
+            <ExternalSourceUpdateCard
+              deleted={true}
+              sources={unseenDeletedSources}
+              on:dismiss={() => {
+                effects.deleteExternalSourceSeenEntry(unseenDeletedSources, user);
+              }}
+            />
+          {/if}
+        </CardList>
+      </div>
+    {/if}
+
+    {#if filteredDerivationGroups.length}
+      {#each Object.keys(mappedDerivationGroups) as sourceType}
+        <Collapse title={sourceType.toString()} tooltipContent={sourceType.toString()} defaultExpanded={true}>
+          {#if mappedDerivationGroups[sourceType]}
+            {#each mappedDerivationGroups[sourceType] as group}
+              <ExternalSourcePanelEntry derivationGroup={group} {user} />
+            {/each}
+          {/if}
+        </Collapse>
+      {/each}
+    {:else}
+      <p>
+        <br />
+        No Derivation Groups Linked To This Plan
```

Comment:
Can you wrap this in a div and add a stellar typography class? Same with the <i> below.